### PR TITLE
Remove issue with technologies patterns search

### PR DESCRIPTION
**Pull request by polubis**

### CODE REVIEW

File: src/modules/admin/dictionaries/components/dictionaries-table/DictionariesTable.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@import 'styles/mixins';
+
+.dictionariesTable {
+  width: 1200px;
+
+  & > table {
+    tr > td:nth-child(3) > div {
```
Comment (Czaroos) [inline src/modules/admin/dictionaries/components/dictionaries-table/DictionariesTable.scss:7]: wouldn't it be easier to simply add a new className? This selector is super confusing and nth-child selector isn't necessarily the best idea, ever

Comment (polubis) [reply src/modules/admin/dictionaries/components/dictionaries-table/DictionariesTable.scss:7]: Adding dedicated classNames which will be available globally or adding suppa not needed - variables to config only for change one cell will me more confusing. Its normal selector.

**Review by i0skar — APPROVED** [review]

**Review by Dyqmin — APPROVED** [review]

File: src/modules/admin/dictionaries/components/dictionaries-table/confirm-dictionary-delete/ConfirmDictionaryDelete.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  const handleDelete = useCallback(async () => {
+    setPending(true);
+
+    try {
+      const callApi = DictionaryKind.TECHNOLOGIES ? deleteTechnology : deletePattern;
+
+      await callApi(data.id);
+
+      history.push(`/app/admin/dictionaries/${kind}`);
+
```
Comment (ajronn) [inline src/modules/admin/dictionaries/components/dictionaries-table/confirm-dictionary-delete/ConfirmDictionaryDelete.tsx:32]: I would remove these offsets, but it is not necessary.

File: src/modules/admin/dictionaries/hooks/useRouteValidation.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+export const useRouteValidation = (kind: DictionaryKind, { replace }: History) => {
+  useEffect(() => {
+    if (!kind) {
+      const url = Url(location).concat(`/${DictionaryKind.PATTERNS}`).value();
+
+      replace(url);
+    } else if (!isValidCategory(kind)) {
+      const url = Url(location).replace(kind, DictionaryKind.PATTERNS).value();
+
+      replace(url);
```
Comment (ajronn) [inline src/modules/admin/dictionaries/hooks/useRouteValidation.ts:19]: same as above

**Review by ajronn — APPROVED** [review]

**Pull request merged**